lib/applications.js

```diff
@@ -100,15 +103,17 @@ exports.updateApplication = async (req, res) => {
     }
     req.body.user_id = req.user.id;
     req.body.event_id = req.event.id;
-    req.body.email = req.user.notification_email;
+    req.body.email = req.user.email;
 
     await sequelize.transaction(async (t) => {
         // Updating application in a transaction, so if mail sending fails, the update would be reverted.
         await req.application.update(req.body, { transaction: t });
 
+        req.application.notification_email = await core.fetchUser(req.body, req.headers['x-auth-token']);
```

Comment:
and this one as well

Reply:
No, since applications can be changed by other users. So that would not work

Reply:
but it also sets other fields from req.user: https://github.com/AEGEE/events/blob/master/lib/applications.js#L96
so either this is wrong too (can be, I don't remember actually) or it can/should be done by user only

Reply:
Looking at https://github.com/AEGEE/events/blob/master/lib/helpers.js#L239 it seems like you can edit applications as EQAC/admins. But in the frontend that is not possible since that is also wrong. I will fix it later and placed a TODO there for now

Reply:
With not possible I mean not possible through buttons, with changing the URL you can go to another application and upon saving that one it will actually change the user it's attached to

Reply:
we should fix this as well probably, agree